src/vm/eetwain.cpp

```diff
@@ -2413,6 +2413,18 @@ unsigned scanArgRegTableI(PTR_CBYTE     table,
     _ASSERTE(*castto(table, unsigned short *)++ == 0xBABE);
 #endif
 
+    bool      isPartialArgInfo;
```

Comment:
A nit, could you please name it hasPartialArgInfo instead (like in "the current frame **has** partial arg info")?